### PR TITLE
[DSCP-207] Refresh balance after successful tx and also periodically

### PR DESCRIPTION
**Pull request by goodsoft**

### CODE REVIEW

**Comment by Martoon-00** [comment]
Ueeee, kaef

File: wallet/exec/Main.hs [hunk 1]
```diff
@@ -54,4 +57,10 @@ initializeEverything serverAddress = do
         uiAction :: IO ()
         uiAction = mkUiAction (knitFaceToUI uiWalletState uiFace walletFace knitFace)
 
-    return uiAction
+        refreshAction :: IO ()
+        refreshAction = forever $ do
+            delay <- randomRIO (15, 45)
```
Comment (Martoon-00) [inline wallet/exec/Main.hs]: Slot duration in production will be 20s, I believe we can make this less.
Poor @ArgentumHeart may just fall asleep while waiting for 40 seconds.

Comment (goodsoft) [reply wallet/exec/Main.hs]: 15..25 then

**Comment by goodsoft** [comment]
Dunno, we have feature freeze and all.
I'm committing everything to `master` unless specifically told to do otherwise.

**Pull request merged**